ui/components/app/wallet-overview/eth-overview.js

```diff
@@ -72,6 +76,8 @@ const EthOverview = ({ className }) => {
   });
   const defaultSwapsToken = useSelector(getSwapsDefaultToken);
 
+  const swapPrimary = !conversionRate && !useNativeCurrencyAsPrimaryCurrency;
```

Comment:
[req] i assumed this was related to swaps, lol. maybe `treatSecondaryCurrencyAsPrimary` ? 😬 naming things is hard

Reply:
> ![image](https://user-images.githubusercontent.com/4448075/122585858-9196a800-d021-11eb-9b4c-5043947c6b28.png)
> Is this file supposed to be here? Seems off

definitely not 😁